packages/@react-aria/slider/intl/el-GR.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Maximum": "Το μέγιστο",
+  "Minimum": "Ελάχιστο"
```

Comment:
Ah, just noticed but these should be lowercase or it breaks when switching to greece locale
```suggestion
  "maximum": "Το μέγιστο",
  "minimum": "Ελάχιστο"
```

Reply:
Not sure why they got switched, but you're right.